tests/test_qcs_backend.py

```diff
@@ -85,19 +85,19 @@ def test_run__parametric_circuits(backend: RigettiQCSBackend):
     assert result.backend_name == backend.configuration().backend_name
     assert len(result.results) == 4
 
-    assert result.results[0].header.name.startswith(f"{circuit1.name}-")
+    # assert result.results[0].header.name.startswith(f"{circuit1.name}-")
     assert result.results[0].shots == 1000
     assert result.get_counts(0).keys() == {"0", "1"}
 
-    assert result.results[1].header.name.startswith(f"{circuit1.name}-")
+    # assert result.results[1].header.name.startswith(f"{circuit1.name}-")
     assert result.results[1].shots == 1000
     assert result.get_counts(1).keys() == {"0", "1"}
 
-    assert result.results[2].header.name.startswith(f"{circuit2.name}-")
+    # assert result.results[2].header.name.startswith(f"{circuit2.name}-")
     assert result.results[2].shots == 1000
     assert result.get_counts(2).keys() == {"0", "1"}
 
-    assert result.results[3].header.name.startswith(f"{circuit2.name}-")
+    # assert result.results[3].header.name.startswith(f"{circuit2.name}-")
```

Comment:
Why were these commented out?

Reply:
It seems the update to Qiskit changes the circuit names upon parameterization. These commented out lines should be fully deleted in the MR pending some tests.